relay-config/src/config.rs

```diff
@@ -706,6 +706,10 @@ struct Http {
     /// During a network outage relay will try to reconnect and will buffer all upstream messages
     /// until it manages to reconnect.
     outage_grace_period: u64,
+    /// Time Relay waits before retrying an upstream request, in milliseconds.
+    ///
+    /// This time is only used before going into a network outage mode.
+    unavailable_upstream_retry_period: u64,
```

Comment:
Maybe it's better to call this interval? 
```suggestion
    unavailable_upstream_retry_interval: u64,
```
Since this is the time Relay will wait to retry again and again, won't it ?